pkgs/applications/editors/emacs/elisp-packages/updater-emacs.nix

```diff
@@ -3,27 +3,30 @@ let
 
   emacsEnv = pkgs.emacs.pkgs.withPackages (epkgs: let
 
-    promise = epkgs.trivialBuild {
+    promise = epkgs.melpaBuild {
       pname = "promise";
-      version = "1";
+      version = "0-unstable-2019-06-07";
+
       src = pkgs.fetchFromGitHub {
-        owner = "bendlas";
-        repo = "emacs-promise";
+        owner = "emacsmirror";
+        repo = "promise";
```

Comment:
I slightly against this change because I do not see any benefit of it and it adds "noise" to `git blame`.